editor/scene_tree_dock.cpp

```diff
@@ -2718,7 +2718,7 @@ void SceneTreeDock::_update_create_root_dialog() {
 				if (l != String()) {
 					Button *button = memnew(Button);
 					favorite_nodes->add_child(button);
-					button->set_text(TTR(l));
```

Comment:
I think this one is impossible to fetch the text to be translated? 
The lines are from a file.